addons/account/report/account_aged_partner_balance.py

```diff
@@ -147,7 +147,8 @@ def _get_partner_move_lines(self, account_type, date_from, target_move, period_l
             partners_amount = {}
             aml_ids = cr.fetchall()
             aml_ids = aml_ids and [x[0] for x in aml_ids] or []
-            for line in self.env['account.move.line'].browse(aml_ids):
+            move_lines = self.env['account.move.line'].browse(aml_ids)
+            for line in move_lines.with_context(prefetch_fields=False):
```

Comment:
You can simply do:
`for line in self.env['account.move.line'].browse(aml_ids).with_context(prefetch_fields=False)`

Reply:
@nim-odoo 

I changed the commit message, I hope it is fine now. I also change the code as you suggests it.

Reply:
I think it should be  `self.env['account.move.line'].with_context(prefetch_fields=False).browse(aml_ids):`